### ISSUE TITLE
Should the argmax parameter come from softmax?

### ISSUE THREAD

**Issue by uranusx86** [dalgu90/resnet-18-tensorflow#8]
the argmax parameter [here](https://github.com/dalgu90/resnet-18-tensorflow/blob/49eb67c3c57258537c0dcbab5cdf2c38bb1af776/resnet.py#L69) directly from logits, is it a bug?
Should the argmax parameter come from softmax?

**Comment by Hugstar** [comment]
@uranusx86 softmax is an order-preserving operation, so the argmax before/after softmax is the same
